app/workers/question_worker.rb

```diff
@@ -0,0 +1,37 @@
+class QuestionWorker
+  include Sidekiq::Worker
+  def perform
+    require 'post_to_slack'
+
+    # First check there is no active question already
+    if Question.where(open: true).none?
+      # get a random unanswered question
+      today = Question.left_outer_joins(:responses).where(responses: {id: nil}).order("RANDOM()").first
+      if today
+        today.open = true
+        today.date_asked = Date.today
+        today.save
+        # puts today.question
+        message = "Good morning! After your cup of :coffee:, here\'s today\'s question:\n>*#{today.question}*"
+        attachments = []
+        # Get all active users and DM them the question
+        allUsers = User.where(active: true)
```

Comment:
This is javascript style, use underscores for ruby, e.g. `all_users`. Check out our ruby styleguide: https://github.com/kohactive/ruby-styleguide#snake-case-symbols-methods-vars